### ISSUE TITLE
Push docs to `gh-pages` branch

### ISSUE THREAD

**Issue by dmcdougall** [manufactured-solutions/MASA#30]
As mentioned in #22, we're not currently making our documentation available without building.

We could push built docs for the latest release up to the gh-pages branch and then they'd be available for free via GitHub pages.

**Comment by nicholasmalaya** [comment]
Cool thought, would be great to add. However, I'm having trouble populating the github.io page. I created a gh-pages branch: https://github.com/manufactured-solutions/MASA/tree/gh-pages with at least a nominal html index. 

However, directing towards:  https://manufactured-solutions.github.io/masa gives 404.

**Comment by nicholasmalaya** [comment]
Alright, after chatting with @dmcdougall I've pushed the latest *.html resulting from the 'masa doc' in the latest build. The results of this are in: http://manufactured-solutions.github.io/MASA/ I've also added a pointer to this on the main masa readme. 

I think this looks pretty good. Would be fun to add a hook that automatically pushes the files from buildbot or travisCI to this branch as an update, but for now the result is static. 

I'm inclined to close this issue.
